docs/source/new-features/index.rst

```diff
@@ -39,6 +39,23 @@ Find out about the details of major features, changes, and deprecations below.
                     </div>
                 </div>
             </div>
+            <div class="grid-card">
+                <div class="content-container">
+                    <div class="header">
+                        Updated Model Registry UI
+                    </div>
+                    <img class="card-image" src="../_static/images/logos/mlflow-logo.svg" alt="MLflow"></img>
+                    <div class="body">
+                        A new opt-in Model Registry UI has been built that includes alias support for self-defined model stages. See more about 
+                        <a href="../model-registry.html#model-registry-workflows">model aliases</a> and 
+                        <a href="../model-registry.html#promoting-an-mlflow-model-across-environments">model promotion across environments</a> in the docs. 
```

Comment:
```suggestion
                        A new opt-in Model Registry UI has been built that uses Aliases and Tags for managing model development. See 
                       more about the new <a href="../model-registry.html#ui-workflow">UI workflow</a> in the docs.
```

Reply:
Here's a suggested edit. I'm removing the model promotion link because it does not yet affect the UI workflow. It will though in the next release.